tests/sentry/rules/test_processor.py

```diff
@@ -246,11 +246,13 @@ class RuleProcessorTestFilters(TestCase):
         "tests.sentry.rules.test_processor.MockFilterFalse",
     )
 
+    def setUp(self):
+        self.event = self.store_event(data={}, project_id=self.project.id)
+        self.event = next(self.event.build_group_events())
```

Comment:
worth changing this variable name to reflect that it's a group event? might be a separate PR due to size of change though with line 53 as well

Reply:
Yup, good idea. Just added.